hvpy/api.py

```diff
@@ -0,0 +1,13 @@
+__all__ = ["fake_api_call"]
+
+
+def fake_api_call():
+    """
+    _summary_
+
+    Returns
+    -------
+    _type_
+        _description_
+    """
```

Comment:
Do these comments get parsed into the docs?

Reply:
They will be, I just created this to ensure the doc build works.